### PR TITLE
2944: ruuTh_ke tum to chal di_e

### PR DESCRIPTION
**Pull request by morosebose**
- Add missing stanza
- Add ISB credits
- Add comment
- Corrections in lyrics

### CODE REVIEW

**Comment by morosebose** [comment]
Among all the usual self-congratulatory blather that marked every post of his, Nani said this when he submitted this song for FITB: 

> In 1994, I made copy of some Anil Biswas songs for a friend of mine, the lovely Lata solo 'rooth ke tum to chal diye' (Jalati Nishaani) being one of them. His father,  Dr Sharad Jog, heard this old favourite of his after many, many years and was mesmerised by it all over again. I can't imagine why he chose to write to Har Mandir Singh 'Hamraaz', requesting from him the lyrics when he could easily have taken them down himself. Probably it is the lyrics-deafness in me that makes me wonder about Dr Jog’s action; it is much more likely that it was God doing His prompting. Hamraaz replied, missing stanza and all, and Dr Jog showed me his letter. Dr Jog then spent considerable time and energy to locate the third stanza in Lata's voice by contacting his musical friends in Pune, Mumbai, Gulf, England and US, but without any luck. I contacted Anil Biswas over phone to get information about the missing stanza and he suggested I contact his friend, Pravin Junnarkar, based in Vadodara. Junnarkar did not have the third stanza, and was himself very curious about it. Later, Anil Biswas told him that he had recorded only two stanzas.

tl;dr, whether this third stanza actually exists is a metaphysical question. But hey, the mythic second verse ("tuufaan hazaaro.n saath li_e") of "puuchho na hame.n ham unake li_e" made it into the ISB despite never having been recorded, so why not.
